x-pack/plugins/maps/public/components/_index.scss

```diff
@@ -1,5 +1,5 @@
 @import './gis_map/gis_map';
-@import './layer_addpanel/layer_addpanel';
+@import './layer_addpanel/source_select/source_select';
```

Comment:
Can you follow the pattern of the imports below where there's an `_index.scss` file in the `source_select` folder that imports the `source_select` file?